test-fixtures/generate-test-db.py

```diff
@@ -33,10 +34,12 @@
 import re
 import subprocess
 import logging
+import inquirer
 
-BUNDLE_ID = 'org.mozilla.ios.Fennec'
```

Comment:
I would say we can keep Fennec scheme for now. Usually when debugging we use that one. Even though we will run tests on physical devices using FennecEnterprise, we will keep the ones running as they are now.

Reply:
ah, okay; makes sense now. That's the default value to use, and I changed it after noticing the scheme bundle id had changed on me somehow even though i'm using the same scheme as i've been using for testing - assumed we needed to update it, but i'll change it back so we're defaulting to `Fennec`.